tcms/settings/common.py

```diff
@@ -208,11 +208,11 @@
 ]
 
 # If you set this to False, Django will not use timezone-aware datetimes.
-USE_TZ = False
+USE_TZ = os.environ.get('KIWI_USE_TZ', False)
```

Comment:
If this value is set then `os.environ.get` will return a string, not a bool. Wrap this around a `bool()` call.

Reply:
good catch. will use  
`USE_TZ = true if os.environ.get('KIWI_USE_TZ') == str(True) else False`
since non-empty string will return true always if using bool('False').

Reply:
if KIWI_USE_TZ is set too False or any non-empty string, `os.envuse iron.get('KIWI_USE_TZ')`  will resolve to True, this is why I use `str(True)` in the code to make sure only "True" is taken as authentic.